### ISSUE TITLE
database should be empty error after migration

### ISSUE THREAD

**Issue by AliMughal169** [InvoiceShelf/InvoiceShelf#203]
Hi, I am using Herd for installation of the project . i have followed all the steps mentioned in https://docs.invoiceshelf.com/developer/laravel-herd.html . but after running "php artisan migrate" command during installation it shows me an error saying "database should be empty" on database config page.

so I tried deleting all the databases and then run installation. but each time i complete database config page it creates new database and changes the path of database in .env file which makes the vite server to restart as a result it gets back to the first page of installation so it repeats again and again.

can someone guide me on this please

PHP : 8.3
Platform: Herd 
OS: windows
Database:SQLite

**Comment by mchev** [comment]
Not sure but i think the step 6 in the documentation is not necessary since the installation process will migrate for you. If you migrate before launching the installation process, you will get errors.

You should just have an empty database (with no table at all) if using mysql ou postgresql. If using sqlite, just make sure there is no database file in the database directory.

**Comment by AliMughal169** [comment]
yes, when i removed all the database files for sqlite it created the database by itself and did migration too. but the problem is that when it changes the path in .env file it makes the vite server to restart and i have to start again from first page of installation

**Comment by Thawshi-Srikanth** [comment]
I also had the same problem where changing the path in the `.env` file caused the Vite server to restart, making me start the installation process from the beginning. I fixed it by ignoring the `.env` files in the watch configuration:

server: {
  watch: {
    ignored: ['**/.env', '**/.env.*'],
  },
}

**Comment by christoferd** [comment]
> so I tried deleting all the databases and then run installation. but each time i complete database config page it creates new database and changes the path of database in .env file which makes the vite server to restart as a result it gets back to the first page of installation so it repeats again and again.

https://docs.invoiceshelf.com/developer/laravel-herd.html

**Step 4:** Instead of `npm run dev` do `npm run build`

**Step 6:** Skip this step
 
Note: later when developing, instead do: npm run dev

Problem now is that if you had issues during the installation you'll see a new error on the next step of the installation process.
"Error CSRF token mismatch."

**Comment by rihards-simanovics** [comment]
Is this still an issue? If so, please let me know I'll re-open this, but I will close this for now as the issue sounds stale.
